Matlab/Default.sublime-keymap

```diff
@@ -0,0 +1,10 @@
+[
+    // Disable auto-pair for single quote when used as transpose operator
+    { "keys": ["'"], "command": "insert_snippet", "args": {"contents": "'"}, "context":
+        [
+            { "key": "selector", "operator": "equal", "operand": "source.matlab" },
+            { "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
+            { "key": "preceding_text", "operator": "regex_contains", "operand": "[\\.)}\\]]$", "match_all": true }
```

Comment:
```suggestion
            { "key": "preceding_text", "operator": "regex_contains", "operand": "[])}.]$", "match_all": true }
```

With little changes, no escapes are needed in the `preceding_text` pattern. Not too important though.